src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.tsx

```diff
@@ -136,8 +125,8 @@ type Props = {
   // Callbacks //
   onChangeProjects?: (val: number[]) => void;
   onUpdateProjects?: (selectedProjects: number[]) => void;
-  onChangeEnvironments?: (environments: Environment[]) => void;
```

Comment:
No `Environment`?

Reply:
The types for this were just wrong. I dropped a console.log() in the handler and it receives strings.